klasses/admin.py

```diff
@@ -104,17 +126,24 @@ class BootcampRunCertificateAdmin(TimestampedModelAdmin):
     ]
     raw_id_fields = ("user",)
 
+    def get_queryset(self, request):
+        return self.model.all_objects.get_queryset().select_related(
```

Comment:
Should we consider adding a doc-string?

Reply:
It's an overridden method and I think it's clear enough what its used for